.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,34 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
```

Comment:
I am curious.
Do we really need it on the release branches?
These are only merged from PRs which are already passing CI.
Imho, this is unnecessary.

Reply:
It's good to have a sanity check.